src/loop_libraries/library_collier.cpp

```diff
@@ -126,15 +132,15 @@ BOOST_PP_SEQ_FOR_EACH(COLLIER_B, (B_ARGS), B_SEQ)
 BOOST_PP_SEQ_FOR_EACH(COLLIER_C, (C_ARGS), C_SEQ)
 BOOST_PP_SEQ_FOR_EACH(COLLIER_D, (D_ARGS), D_SEQ)
 
-void Collier::A(std::array<std::complex<double>, 1>& a, A_ARGS) noexcept
+void Collier::A(std::array<std::complex<double>, A_N>& a, A_ARGS) noexcept
```

Comment:
Would you mind replacing the `std::array<...>` types by `Loop_library_interface::Acoeff_t` (and similarly for the `B`, `C` and `D` functions) here and everywhere else?

Reply:
Yes, this would be much better.